src/saturn_engine/worker/services/loggers/logger.py

```diff
@@ -214,8 +217,8 @@ def on_pipeline_executed(
             try:
                 yield
                 self.logger.debug("Executed pipeline", extra=extra)
-            except Exception:
-                self.logger.exception("Failed to executed pipeline", extra=extra)
+            except:
```

Comment:
As stated above, don't do naked `except:`. I would keep logging the exception at least in debug.